### PR TITLE
Adding Retail to relnotes

### PR DESCRIPTION
**Pull request by Loquacity**
@mcalmer for visibility.

### CODE REVIEW

File: modules/ROOT/pages/release-notes-version-4.0.adoc [hunk 1]
```diff
@@ -119,10 +119,11 @@ All {productname} books have been reworked into a new structure.
 * Installation Guide (Requirements, supported platforms, installation methods, etc)
 * Client Configuration Guide (Configuring and connecting clients to {productname})
 * Upgrade Guide (Migrate and update clients and {productname})
-* Reference Manual (Comprehensive guide to the {webui})
+* Reference Guide (Comprehensive guide to the {webui})
 * Administration Guide (Maintenance and administration tasks in {susemgr})
 * Salt Guide (A comprehensive guide to Salt for system administrators)
-* Architecture (Details on components, salt/traditional architectures, and diagrams)
+* Retail Guide (A guide to using {smr})
+* Architecture Guide (Details on components, salt/traditional architectures, and diagrams)
```
Comment (keichwa) [inline modules/ROOT/pages/release-notes-version-4.0.adoc]:
```suggestion
* Architecture Guide (Details on components, Salt and traditional architectures, and diagrams)
```

**Comment by jcayouette** [comment]
I believe almost in every other location we are using **Reference Manual**? I am of course open to changing this, but I think that the reference documentation is really less of a guide and more of a manual.

**Comment by keichwa** [comment]
> I believe almost in every other location we are using **Reference Manual**? I am of course open to changing this, but I was under the impression that the reference documentation is really less of a guide and more of a manual.

Yes, you are right, agreed, +1.

The official name used to be just "Reference", manual (or guide) was not part of the title.  Of course, I'll leave it to you as native speakers to select a good title.

**Comment by Loquacity** [comment]
> I believe almost in every other location we are using **Reference Manual**? I am of course open to changing this, but I was under the impression that the reference documentation is really less of a guide and more of a manual.

I have no idea what the difference between a Guide and a Manual might be. I was just looking at the parallelism (or lack thereof).

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> I have no idea what the difference between a Guide and a Manual might
> be.

Interesting ;)  A former SUSE employee from the UK (GB) made a big fuss
about the differences of Guide and Manual...  He unfortunately passed
away several years ago.

> I was just looking at the parallelism (or lack thereof).

Fair enough!

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by Loquacity** [comment]
> I have no idea what the difference between a Guide and a Manual might be.
> Interesting ;) A former SUSE employee from the UK (GB) made a big fuss about the differences of Guide and Manual... He unfortunately passed away several years ago.

I'm sorry to hear that :( I'd love to know what his reasoning was!

In an effort to enlighten myself, I have posed the question to the WtD Slack channel. I shall report back!

**Comment by keichwa** [comment]
On 2019-05-01 02:44, Lana Brindley wrote:

> In an effort to enlighten myself, I have posed the question to the WtD
> Slack channel. I shall report back!

Thanks!  Could be interesting.  The more I try remembering it, I've the
feeling that the discussion mostly was about handbook vs. manual ;)

I think he wanted to use handbook for our big combined SUSE Linux
Installation and Administration Guide, and his co-translators said that
this is a Germanism.  (At that time, we wrote our books in German, and
our translators had to translate to English and all the other languages.
Funny times!)

-- 
Karl Eichwalder

**Comment by Loquacity** [comment]
Well, that was certainly interesting, but I am no further enlightened. It basically boiled down to consistency, which might as well be the hallmark of tech writers everywhere ("do whatever you like! just do it the same everywhere!"). I have attached the screenshots for your own entertainment:

GuidevManual_1
GuidevManual_2
GuidevManual_3

**Comment by keichwa** [comment]
> Well, that was certainly interesting, but I am no further enlightened. It basically boiled down to consistency, which might as well be the hallmark of tech writers everywhere ("do whatever you like! just do it the same everywhere!"). I have attached the screenshots for your own entertainment:

Thanks for sharing! :-) Maybe, Joseph was not that wrong.  Reference is different, thus let's tag it differently.  And the others are Guides.

**Comment by Loquacity** [comment]
@jcayouette I need your approving review before I can merge. Are you happy with this?

**Pull request merged**